### ISSUE TITLE
Interactive embeds are unstable

### ISSUE THREAD

**Issue by thexeos** [twitchdev/issues#157]
As per https://discuss.dev.twitch.tv/t/twitch-embedded-player-updates-in-2020/23956 we are using https://embed.twitch.tv/embed/v1.js to create interactive embeds on our (HTTPS) site, so the expectation is that it should just work. We do *not* pass the `parent` array to the constructor and the site itself is not embeddable (and hence is the topmost frame).

The `parent` parameter (and page href? that seems unwarranted) is being passed to the embed endpoint, and it works, until it doesn't. We've been getting user reports and screenshots of the "Whoops! This embed is misconfigured." error since June 10th.

In all cases, reloading the page and trying again or restarting the browser and trying again solved the "issue". This happens across the board on desktop and mobile browsers. I have no reproduction steps or any additional info, as these seem absolutely random. The most recent report was from an hour ago or so.

**Comment by BarryCarlyon** [comment]
This should be on the developers forums rather than the github.

Whats your website/page with an embed on it?

**Comment by thexeos** [comment]
The domain is `gamingtribe.com`

I got here from the link in this post https://discuss.dev.twitch.tv/t/twitch-embedded-player-updates-in-2020/23956/28 but I'm happy to move this to dev forums.

**Comment by BarryCarlyon** [comment]
I don't know if your problem right now is a bug or an implementation issue.

What you currently have sounds like a implementation issue. Which is why we have closed the thread you link to in favor of

https://discuss.dev.twitch.tv/t/the-great-big-im-using-wix-and-i-dont-know-how-to-fix-my-embed/26297

There are no broken embeds on the page you have linked to. There are no embeds at all.

**Comment by thexeos** [comment]
I referenced that thread to show the reason why this was posted here and not on dev forums.

The implementation follows this guide:
https://dev.twitch.tv/docs/embed/video-and-clips#interactive-frames-for-live-streams-and-vods

But you are right. This is an implementation issue on our end. I got a screenshot of what the users are seeing and it was in fact a chat window which is optional and appears to the side of the player and is constructed explicitly with no `parent` attribute. This is now resolved.

However this issue got mixed with another, unrelated one, which results in a blank embedded player iframe due to a JS error that is not caught by the Twitch Player code.

![image](https://user-images.githubusercontent.com/2260539/87098954-6c5dbf00-c240-11ea-9642-5a981936a470.png)

Which in turn is caused by https://stackoverflow.com/a/44069936 (non-default, strict privacy setting in Chrome, which is auto-enabled in Incognito Mode).

The only thing Twitch devs can do here is handle the error gracefully and not crash the main player embed thread.
